com.unity.render-pipelines.universal/Shaders/Utils/TileDeferred.shader

```diff
@@ -5,8 +5,11 @@ Shader "Hidden/Universal Render Pipeline/TileDeferred"
     #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
     #include "Packages/com.unity.render-pipelines.universal/Shaders/Utils/Deferred.hlsl"
 
-    // XR not supported in 2020.1 preview
+    #if defined(USING_STEREO_MATRICES) && (defined(_POINT) || defined(_SPOT) || defined(_DIRECTIONAL))
+    #define XR_MODE 1
```

Comment:
Is it possible to use USING_STEREO_MATRICES instead? 
I would prefer to use pre-existing XR shader defines instead of defining a new one.